watchers/configmap.go

```diff
@@ -56,19 +57,23 @@ func (w *ConfigMapWatcher) Start(ctx context.Context) {
 }
 
 // watchChange consumes watch events and returns on change.
-func (w *ConfigMapWatcher) watchChange(ctx context.Context) error {
+func (w *ConfigMapWatcher) watchChange(ctx context.Context, waitPeriod int) error {
```

Comment:
Same not about `waitPeriod` here:
> It's probably clearer for waitPeriod to be a time.Duration, otherwise it's easy for people to assume the wrong unit